flow-server/src/main/java/com/vaadin/flow/server/frontend/TaskGenerateHilla.java

```diff
@@ -24,6 +24,16 @@
  * For internal use only. May be renamed or removed in a future release.
  */
 public interface TaskGenerateHilla extends FallibleCommand {
+    /**
+     * Configures the task by passing it some parameters.
+     *
+     * @param projectDirectory
+     *            the project root directory. In a Maven multi-module project,
+     *            this is the module root, not the main project one.
+     * @param buildDirectoryName
+     *            the name of the build directory (i.e.&nbsp;"build" or
```

Comment:
I think HTML space entity is not needed
```suggestion
     *            the name of the build directory (i.e. "build" or
```

Reply:
Actually, I don't remember if @param description are shown in short format somewhere, so I put that there just in case.

Reply:
Sorry, I don't understand why using a single `&nbsp;`. Can you please elaborate your previous answer?

Reply:
I don't think it's needed as only method descriptions are truncated to the first sentence in Javadoc Summaries. I think that parameter description are never truncated.

Reply:
Thanks for explanation